src/customizations/volto/components/manage/Contents/ContentsUploadModal.jsx

```diff
@@ -196,10 +196,12 @@ class ContentsUploadModal extends Component {
     // as per agid guidelines, files cannot be uploaded in modulistica folder inside ct servizio
     // show restraint and hide upload button when page is called modulistica and when there's a restraint
     // this is enough to identify this only case bc even if another page is called 'modulistica', it will have 'File'
-    // as addable type
+    // and "Modulo" as addable type
     const showFileRestraint =
       this.props.pathname.includes('modulistica') &&
```

Comment:
```suggestion
      getBaseUrl(this.props.pathname).match(/\/modulistica\/?$/) &&
```

Questo dovrebbe controllare in maniera più strict se sei in una cartella che ha proprio l'id fisso modulistica. Dato che vogliamo impedire che vengano inseriti File nella modulistica dei servizi, potremmo anche avere una espressione regolare ancora più strict, del tipo `/\/servizi\/.*\/modulistica\/?$/`

Reply:
direi allora di mettere quello più specifico